### PR TITLE
Create WD AV Signature and Platofrm Version.txt

### PR DESCRIPTION
**Pull request by anthonws**

### CODE REVIEW

File: General queries/WD AV Signature and Platofrm Version.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+//Author: António Vasconcelos
+//Twitter: https://twitter.com/anthonws
+// This query provides you the latest signature and platform (MoCamp) for Windows Defender AV
+FileCreationEvents
+| where InitiatingProcessCommandLine contains "MpSigStub.exe"
```
Comment (meni-braun) [inline General queries/WD AV Signature and Platofrm Version.txt]: Performance: please make sure to filter by time:
`let startDate = ago(1d);`
and later on both queries on `FileCreationEvents` filter by it:
`and EventTime > startDate`

Comment (meni-braun) [inline General queries/WD AV Signature and Platofrm Version.txt]: Performance: prefer `has` over `contains` everywhere it is possible

File: General queries/WD AV Signature and Platofrm Version.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+| extend NewVersion=tostring(split(InitiatingProcessCommandLine, " ")[4]) 
+| summarize arg_max(NewVersion, *) by ComputerName
+| project ComputerName , NewVersion
+| join (FileCreationEvents
+        | where FileName == "MsMpEng.exe"
+        | where FolderPath contains @"C:\ProgramData\Microsoft\Windows Defender\Platform\"
```
Comment (meni-braun) [inline General queries/WD AV Signature and Platofrm Version.txt]: Performance: I think `startswith` can be applied here. For string operators `contains` should be the last option, perf-wise

File: General queries/WD AV Signature and Platofrm Version.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
+//Author: António Vasconcelos
+//Twitter: https://twitter.com/anthonws
+// This query provides you the latest signature and platform (MoCamp) for Windows Defender AV
+FileCreationEvents
+| where InitiatingProcessCommandLine contains "MpSigStub.exe"
+| extend NewVersion=tostring(split(InitiatingProcessCommandLine, " ")[4]) 
+| summarize arg_max(NewVersion, *) by ComputerName
```
Comment (meni-braun) [inline General queries/WD AV Signature and Platofrm Version.txt]: why not by `EventTime`?
i.e. `arg_max(NewVersion, EventTime)`

File: General queries/WD AV Signature and Platofrm Version.txt (hunk 1 above)
Comment (anthonws) [reply General queries/WD AV Signature and Platofrm Version.txt]: Agreed.

File: General queries/WD AV Signature and Platofrm Version.txt (hunk 2 above)
Comment (anthonws) [reply General queries/WD AV Signature and Platofrm Version.txt]: Agreed.

File: General queries/WD AV Signature and Platofrm Version.txt (hunk 3 above)
Comment (anthonws) [reply General queries/WD AV Signature and Platofrm Version.txt]: Agreed.

**Comment by anthonws** [comment]
Please see changes.

Thanks!

**Pull request merged**